TwitchPlaysAssembly/Src/Commands/ModuleCommands.cs

```diff
@@ -76,18 +76,19 @@ public static void BombTurnAroundCancel(TwitchModule module)
 
 	public static IEnumerator Show(TwitchModule module, object yield)
 	{
-		IEnumerator focusCoroutine = module.Bomb.Focus(module.Selectable, module.FocusDistance, module.FrontFace);
+		string[] abortHighlightIds = { "lookLookAway" };
+		IEnumerator focusCoroutine = module.Bomb.Focus(module.Selectable, module.FocusDistance, module.FrontFace, abortHighlightIds.Contains(module.BombComponent.GetModuleID()) ? false : true);
 		while (focusCoroutine.MoveNext())
 			yield return focusCoroutine.Current;
 
 		yield return new WaitForSeconds(0.5f);
 		yield return yield is float delay ? new WaitForSecondsWithCancel(delay, false, module.Solver) : yield;
 		if (CoroutineCanceller.ShouldCancel)
 		{
-			module.StartCoroutine(module.Bomb.Defocus(module.Selectable, module.FrontFace));
+			module.StartCoroutine(module.Bomb.Defocus(module.Selectable, module.FrontFace, abortHighlightIds.Contains(module.BombComponent.GetModuleID()) ? false : true));
 			yield break;
 		}
-		IEnumerator defocusCoroutine = module.Bomb.Defocus(module.Selectable, module.FrontFace);
+		IEnumerator defocusCoroutine = module.Bomb.Defocus(module.Selectable, module.FrontFace, abortHighlightIds.Contains(module.BombComponent.GetModuleID()) ? false : true);
```

Comment:
GitHub won't let me make a suggestion, but I think this code could be significantly simplified:
```csharp
		bool select = !module.BombComponent.GetModuleID().EqualsAny("lookLookAway");
		IEnumerator focusCoroutine = module.Bomb.Focus(module.Selectable, module.FocusDistance, module.FrontFace, select);
		while (focusCoroutine.MoveNext())
			yield return focusCoroutine.Current;

		yield return new WaitForSeconds(0.5f);
		yield return yield is float delay ? new WaitForSecondsWithCancel(delay, false, module.Solver) : yield;
		if (CoroutineCanceller.ShouldCancel)
		{
			module.StartCoroutine(module.Bomb.Defocus(module.Selectable, module.FrontFace, select);
			yield break;
		}
		IEnumerator defocusCoroutine = module.Bomb.Defocus(module.Selectable, module.FrontFace, select);
```